js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
Running `bundle exec jekyll serve` directly (with the `--verbose` flag) shows it getting hung after:
```
   GitHub Metadata: Initializing...
        Generating: Jekyll::GitHubMetadata::GHPMetadataGenerator finished in 0.001932 seconds.
        Generating: JekyllOptionalFrontMatter::Generator finished in 0.091768 seconds.
        Generating: JekyllReadmeIndex::Generator finished in 24.729063 seconds.
```
This is true on `master` as well. Hm. I wonder what broke. This worked for me at one point.